pkg/network/protocols/postgres/model_linux.go

```diff
@@ -52,21 +53,10 @@ func (e *EventWrapper) ConnTuple() types.ConnectionKey {
 	}
 }
 
-// getFragment returns the actual query fragment from the event.
-func (e *EbpfTx) getFragment() []byte {
-	if e.Original_query_size == 0 {
-		return nil
-	}
-	if e.Original_query_size > uint32(len(e.Request_fragment)) {
-		return e.Request_fragment[:len(e.Request_fragment)]
-	}
-	return e.Request_fragment[:e.Original_query_size]
-}
-
```

Comment:
revisiting it, the function does not necessarily need to be moved
you can change it the implementation to get `e *EbpfTx` instead of being an module function

Reply:
I agree, will go for that option